spec/system/casework/assigning/assign_an_application_to_myself_spec.rb

```diff
@@ -3,6 +3,9 @@
 RSpec.describe 'Assigning an application to myself' do
   include_context 'with an existing application'
   let(:assign_cta) { 'Assign to your list' }
+  let(:banner_text) do
+    "You must be allocated to the Cat 1 work queue to review this application\nContact your supervisor to arrange this"
```

Comment:
Is it "Cat 1" or "CAT1"? If its an acronynm we should keep it as caps I think.

Reply:
It's a acronym so will change to caps